### PR TITLE
EN-603 footer

### PR DESCRIPTION
**Pull request by antictor**
Adds a footer component to the docs site designed to work on desktop and mobile.

Tested all the way back to Chrome 38 with no issues.

Screenshot 2019-05-31 at 13 29 26

Simulator Screen Shot - iPad (6th generation) - 2019-05-31 at 13 29 03

Simulator Screen Shot - iPhone Xʀ - 2019-05-31 at 13 29 01

### CODE REVIEW

**Review by 98dfsj — APPROVED** [review]

File: packages/docs-site/src/components/presenters/footer/footer.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  line-height: 1.5;
+  margin: spacing(1) 0;
+}
+
+.rn-footer__copyright {
+  white-space: nowrap;
+}
+
+.rn-footer__divider {
+  border:0;
```
Comment (m7kvqbe1) [inline packages/docs-site/src/components/presenters/footer/footer.scss]:
```suggestion
  border: 0;
```

Comment (m7kvqbe1) [reply packages/docs-site/src/components/presenters/footer/footer.scss]: Pesky little whitespace from the last PR.

Comment (antictor) [reply packages/docs-site/src/components/presenters/footer/footer.scss]: darn..

**Review by m7kvqbe1 — APPROVED** [review]

**Comment by 09txzS1qs** [comment]
Do the "Privacy policy" I "Contact" I "Feedback" links become hyperlinks on hover? Can't tell from a static screenshot.

**Pull request merged**